roles/cloud_setup/tasks/cvmfs_client.yml

```diff
@@ -0,0 +1,55 @@
+---
+# Setup of the CernVM-File system (CVMFS) and configure so that the reference
```

Comment:
I like this a lot. For now I think this is OK here, but what do you think about making the CVMFS specific parts a standalone role?